### ISSUE TITLE
code review

### ISSUE THREAD

**Issue by ashwintiwari88** [ayrtoncn/chatbot#1]
[+] separated dev and prod dependencies
[+] tested all flow scenario

[-] all in one file
[-] no linting / code style
[-] not use db, just saving as object

[+-] not so good flow, the bot only started & works if clicking GET_STARTED (since it only the time creating users). users is saved based on sender id and then it will check the state and update each time the state based on the response

[+-] bot work, but it show the wrong birthday calculation if its months behind

@ayrtoncn  , this is our teams review.

Unless you can make a >50% improvement in the submission , we will unfortunately not be able to get you to the next and final round.

hope this helps!

**Comment by ayrtoncn** [comment]
noted @ashwintiwari88

**Comment by ashwintiwari88** [comment]
@ayrtoncn so will you be making a re-submission ?

**Comment by ayrtoncn** [comment]
Yes. sorry for the late reply, I already re-submit my code and made an improvement except for 
[-] all in one file
[-] not use db, just saving as object
Thank you
@ashwintiwari88

**Comment by ashwintiwari88** [comment]
@ayrtoncn after second review , it still did not meet our internal team vote. wish you best of luck !
